lib/activerecord-import/import.rb

```diff
@@ -592,7 +594,7 @@ def import_without_validations_or_callbacks( column_names, array_of_attributes,
 
     private
 
-    def set_ids_and_mark_clean(models, import_result)
+    def set_attributes_and_mark_clean(models, import_result, timestamps)
```

Comment:
I try not to predict the future and I just find defaults as a nice way to document a method's parameters...but you're right 😉